docs/user/aws/install.md

```diff
@@ -33,11 +36,6 @@ INFO Access the OpenShift web-console here: https://console-openshift-console.ap
 INFO Login to the console with user: kubeadmin, password: XXXX
 ```
 
-This creates an encrypted AMI for the bootstrap and control-plane machines.
-The encrypted AMI is [copied][encrypted-copy] from the AMI configured in the control-plane machine-API provider spec, which is RHCOS by default.
-The encryption uses the default EBS key for your target account and region (`aws kms describe-key --key-id alias/aws/ebs`).
-The encrypted AMI is deregistered by `destroy cluster`.
```

Comment:
It doesn't really matter to me, but for context, the reason I use one-sentence-per-line is that it allows you to edit with sentence granularity without re-wrapping and still get clean diffs even without using `--word-diff=color` or similar.

Reply:
Actually, I don't understand why you moved this creation-time discussion from the "Create Cluster" section to the "Running Cluster" section.  Can you document your motivation?

Reply:
It has more value in the EC2 section about your running cluster for folks inspecting their account and what has happened after the fact. Since we call out the running instances (by count with a picture) they may wonder "where did this unique AMI come from it is running"? It goes along with some of the other explanations we chase with as well. 

The Create Cluster section just has the whole IPI output, but no explanation of all  wizardry under the covers, calling this particular wrinkle out there seemed a bit awkward to me there.

Reply:
> It has more value in the EC2 section about...

I've pushed a comment with (mostly) this wording into the commit message with bdae4f939 -> 39a926a91.